packages/aws-cdk-lib/aws-ec2/lib/vpc.ts

```diff
@@ -1927,6 +2126,34 @@ export class Subnet extends Resource implements ISubnet {
     this._internetConnectivityEstablished.add(route);
   }
 
+  /**
+   * Create a default IPv6 route that points to a passed IGW.
+   *
+   * @param gatewayId the logical ID (ref) of the gateway attached to your VPC
+   */
+  public addIpv6DefaultInternetRoute(gatewayId: string) {
+    this.addRoute('DefaultRoute6', {
+      routerType: RouterType.GATEWAY,
+      routerId: gatewayId,
+      destinationIpv6CidrBlock: '::/0',
+      enablesInternetConnectivity: true,
+    });
+  }
```

Comment:
Also add function for addIpv6Nat64Route, with 64:ff9b::/96 needs to route to a NAT gateway (same as for IPv4, i.e. either one per zone, a single gateway, or round robin). Adding NAT64 allows IPv6-only machines to access IPv4-only resources via NAT (works the same as an IPv4 NAT gateway)